### DISCUSSION TITLE
Horizontal scrolling options: sensitivity and invert

### DISCUSSION THREAD

**Discussion by zorazor** [AppGround-io/bluetooth-keyboard-and-mouse-support#126] [Ideas]
**Is your feature request related to a problem? Please describe.**
1. Horizontal scrolling is too slow.
2. Horizontal scrolling should be invertible.

**Describe the solution you'd like**
Add a separate sensitivity slider and an invert checkbox, please.

**Comment by Genxster1998** [comment]
+1

**Comment by Quimap** [comment]
Hi,

I need to scroll often sideward `<---   --->`

Please can you help?

**Comment by zorazorr** [comment]
Hello, I'm the op.

The problem was reported 1 year ago. I cross my fingers at every update :cry:
Are there maybe technical limitations to adjusting the horizontal scrolling?

**Comment by zorazorr** [comment]
@AppGround-io 
Are there technical problems that make it difficult or impossible to adjust the horizontal scroll speed?
I work on two 21:9 monitors, but occasionally I still have to scroll horizontally. This would make the work flow much more comfortable.
